idunn/blocks/wikipedia.py

```diff
@@ -180,14 +180,23 @@ def with_cache(*args, **kwargs):
             otherwise we bypass it
             """
             if cls._connection is not False:
-                value_stored = cls._connection.get(key)
-                if value_stored is not None:
-                    result = json.loads(value_stored.decode('utf-8'))
+                try:
+                    value_stored = cls._connection.get(key)
```

Comment:
i'd prefer a `get` / `set` method in the class to hide this

Reply:
done